manual/development-process/production-deployment.md

```diff
@@ -0,0 +1,178 @@
+# Deploy code to Production
+
+Transfer Digital Records uses the practice of [Continuous Integration]: developers merge their work into a repo's main
+branch regularly. The repo is configured to only allow changes to be merged if the checks pass, such as unit tests, code
+linters and security checks. These checks are currently run on a [Jenkins instance][tdr-jenkins-intg].
+
+We also perform [Continuous Delivery]. The main branch of any repo is always deployable, and can be deployed to
+Production multiple times a day if necessary. The [end-to-end tests] which run on Integration and Staging help us catch
+regression issues before code is deployed to Production. Deploying frequent changes to Production means that each
+deployment is small, which makes deployments less risky and easier to fix.
+
+TDR is made up of [multiple services][tdr-architecture], each of which are deployed separately. Infrastructure is also
+defined in Terraform repos, and is deployed to each environment in the same way.
+
+## When to deploy
+
+You should deploy to Staging and Production as soon as possible after merging the change and checking it on Integration.
+This means that you can deploy your own change. If you leave your change undeployed and someone else deploys it at the
+same time as theirs, and there failures, it will be harder for them to debug.
```

Comment:
Missing word "...and there **are** failures"

Reply:
Fixed, thanks.